global/tls_config.go

```diff
@@ -32,7 +32,17 @@ type TLSConfig struct {
 
 func DefaultTLSConfig() *TLSConfig {
 	// please refer to /config/tls_config.go NewTLSConfigBuilder
-	return nil
+	return &TLSConfig{}
```

Comment:
To avoid panic, I changed the DefaultTLSConfig function to return a new instance of TLSConfig instead of nil. This should be fine, right?

Reply:
Well, based on the CI results, it seems this can't be changed.